### PR TITLE
Fix/create position issue

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/utils/swap-utils.ts [hunk 1]
```diff
@@ -38,7 +38,9 @@ export function makeSwapFeeTierByTickSpacing(
 }
 
 export function priceToTick(price: number | bigint) {
+  console.log("🚀 ~ priceToTick ~ price:", price);
```
Comment (jinoosss) [inline packages/web/src/utils/swap-utils.ts:41]: Please remove console log.

File: packages/web/src/utils/swap-utils.ts [hunk 2]
```diff
@@ -38,7 +38,9 @@ export function makeSwapFeeTierByTickSpacing(
...
   const logPrice = Math.log(Number(price));
+  console.log("🚀 ~ priceToTick ~ logPrice:", logPrice);
```
Comment (jinoosss) [inline packages/web/src/utils/swap-utils.ts:43]: Please remove console log.

File: packages/web/src/utils/swap-utils.ts [hunk 3]
```diff
@@ -229,6 +231,8 @@ export function getDepositAmountsByAmountA(
   }
 
   const currentPriceX96 = tickToSqrtPriceX96(priceToTick(currentPrice));
+  console.log("🚀 234234234 ~ currentPriceX96:", currentPriceX96);
+  console.log("🚀 234234234 ~ currentPrice:", currentPrice);
```
Comment (jinoosss) [inline packages/web/src/utils/swap-utils.ts:235]: Please remove console log.

File: packages/web/src/hooks/pool/use-select-pool.tsx [hunk 4]
```diff
@@ -481,6 +481,7 @@ export const useSelectPool = ({
     if (num === 0) {
       const { minPrice } = SwapFeeTierMaxPriceRangeMap[feeTier || "NONE"];
       setMinPosition(minPrice);
+      return;
```
Comment (jinoosss) [inline packages/web/src/hooks/pool/use-select-pool.tsx:484]: Is this line a problem?

**Review by jinoosss — COMMENTED** [review]
LGTM ! Only clear console logs.

File: packages/web/src/hooks/pool/use-select-pool.tsx (hunk 4 above)
Comment (phucvarmeta) [reply packages/web/src/hooks/pool/use-select-pool.tsx:484]: Yep, this prevent this function from executing next lines of code

**Review by jinoosss — APPROVED** [review]

**Pull request merged**